public/app/features/dashboard-scene/serialization/layoutSerializers/utils.ts

```diff
@@ -71,6 +72,9 @@ export function buildVizPanel(panel: PanelKind, id?: number): VizPanel {
     seriesLimit: config.panelSeriesLimit,
     $data: createPanelDataProvider(panel),
     titleItems,
+    headerActions: new VizPanelHeaderActions({
```

Comment:
Don't think of this as TODO now, but I think eventually we may want to consider the same pattern as `titleItems`, so that header actions are scene objects and the rendering is performed from the VizPanel. However, given we have single action now, let's stilck with this.